src/methods/malApi/user/index.ts

```diff
@@ -41,13 +41,15 @@ export class MalUser {
     name: string = "@me",
     fields?: AnimeFields<T> | null,
     listStatusFields?: AnimeListStatusFields<S> | null,
+    includeNsfw?: boolean,
     args?: { status?: string; sort?: string; limit?: number; offset?: number }
```

Comment:
I think that this should be treated like other args, and be part of args object.
For example if someone wants to set only `sort` arg, they should not be forced to supply unrelated `nsfw` flag, being part of args object allows this.